ServerCore/Areas/Identity/Pages/Account/Edit.cshtml.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+
+namespace ServerCore.Areas.Identity.Pages.Account
```

Comment:
Can you add a comment here and on the other account settings page warning us to keep them both in sync?

Reply:
done